### PR TITLE
50 bookings handling

### PR DESCRIPTION
**Pull request by any1-source**
added BookingsResource handling:
@Post
@Put (not implementable, because @Put from BookingResource is missing)
@Delete
@Get

### CODE REVIEW

File: backend/src/main/java/de/cae/XYFleet/ressource/BookingResource.java [hunk 1]
```diff
@@ -8,50 +8,42 @@
...
         return result;
     }
-
-    @Post()
-    public void createBooking() {
+    @Override
+    @Put()
+    public String createEntity() {
         checkInRole(ROLE_USER);
+        return null;
```
Comment (N0ah-S) [inline backend/src/main/java/de/cae/XYFleet/ressource/BookingResource.java:40]: I'll just assume you are aware of this everywhere else

File: backend/src/main/java/de/cae/XYFleet/ressource/BookingsResource.java [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
     public String toString() {
         isInRole(XYAuthorizer.ROLE_SECURITY);
-        Result<org.jooq.Record> result = dslContext.select().from(BOOKINGS).fetch();
+        Result<Record> result = dslContext.select().from(BOOKINGS).fetch();
         return result.formatJSON(jSONFormat);
     }
     @Put
-    public String createBookings(){
+    public String createEntity(){
         isInRole(XYAuthorizer.ROLE_ADMIN);
```
Comment (N0ah-S) [inline backend/src/main/java/de/cae/XYFleet/ressource/BookingsResource.java:24]: I'll take this as a yes

File: backend/src/test/java/VehicleResourceTest.java [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.sql.Date;
+
+import static de.cae.XYFleet.authentication.XYAuthorizer.*;
+import static de.cae.XYFleet.authentication.XYAuthorizer.ROLE_ADMIN;
+import static org.jooq.codegen.XYFleet.Tables.*;
+
+public class VehicleResourceTest extends EntryResourceTest {
+    public static int INSURANCEID;
```
Comment (N0ah-S) [inline backend/src/test/java/VehicleResourceTest.java:18]: The convention is usually to separate all CAPS WORDS with underscores
=> INSURANCE_ID

**Review by N0ah-S — APPROVED** [review]
Kinda okay, I'm unable to run the tests/compile the code - but it looks alright. maybe actually rename the two CONSTANTSWITHOUTUNDERSCORES.
LGTM afterwards.

File: backend/src/main/java/de/cae/XYFleet/ressource/BookingResource.java (hunk 1 above)
Comment (any1-source) [reply backend/src/main/java/de/cae/XYFleet/ressource/BookingResource.java:39]: well yes

**Pull request merged**